MarkedMap.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "Map.h"
+#include <string>
+#include <vector>
+#include <fstream>
+#include <list>
+
+class MarkedMap: Map
+{
+    private:
+     std::list<std::pair<std::string,std::pair<int,int>>> monsters;
```

Comment:
Ez felesleges ha getnél oldod meg!

Reply:
Ezt is légyszíves fejtsd ki hogy mire gondoltál, nem pontosan értem

Reply:
Ha a get függvényen belül keresed meg a szörnyek helyét, akkor feleslegessél válik

Reply:
ohh igen, ha átirjuk akkor valóban.